lavamoat/build-system/policy.json

```diff
@@ -957,6 +957,70 @@
         "gulp>gulp-cli>isobject": true
       }
     },
+    "@lavamoat/allow-scripts>@npmcli/run-script>node-gyp>npmlog": {
```

Comment:
not sure we want to add the diff of this file! I believe it's platform dependent. Is that right @kumavis @brad-decker @legobeat?